package.json

```diff
@@ -86,7 +86,7 @@
     "redux-observable": "^0.15.0"
   },
   "peerDependencies": {
-    "@folio/stripes": "^4.0.0",
+    "@folio/stripes": "^5.0.0",
```

Comment:
A major version bump in a peer-dep implies a major version bump to this package as well, so this should be `3.0.0` on line 3.